pkg/controllers/capr/dynamicschema/dynamicschema.go

```diff
@@ -117,9 +136,11 @@ func addMachineTemplateSchema(name string, specSchema *schemas.Schema, allSchema
 		ID: id,
 		ResourceFields: map[string]schemas.Field{
 			"spec": {
-				Type: templateSpecSchemaID,
+				Type:        templateSpecSchemaID,
+				Description: "Specification of the machines in the template, generated from the pool configuration.",
 			},
 		},
+		Description: fmt.Sprintf("%s is a CAPI infrastructure machine template for the rancher %s infrastructure provider.", convert.Capitalize(id), name),
```

Comment:
```suggestion
		Description: fmt.Sprintf("%s is a CAPI infrastructure machine template for the %s infrastructure provider.", convert.Capitalize(id), name),
```

IIUC name is going to be something like `digitalocean`, in which case this wording makes more sense IMO

Reply:
I left `rancher` to make it clear that it's not some other infrastructure capi provider for the given cloud provider. Do  you think it's already obvious from context?